webpack.config.js

```diff
@@ -35,12 +35,12 @@ const config = {
     rules: [
       {
         enforce: "pre",
-        test: /\.js$/,
+        test: /\.jsx?$/,

```

Comment:
you should include the test regex in the production config too.

Reply:
done :)